### ISSUE TITLE
Can I use dlib to finetune my model ?

### ISSUE THREAD

**Issue by minmummax** [davisking/dlib#968]
I am wondering whether can I use dlib to train the last two layer of the classic networks to finetune my own datasets?

**Comment by davisking** [comment]
Yes, you can finetune, freeze layers, and all the usual stuff with dlib.
The entire API is documented in detail on the dlib website and there are
introductory DNN examples that walk you through much of it.

**Comment by minmummax** [comment]
oh really ?but i cannot find the good examples to guide me ?so sad,how can i find good examples to make it through

**Comment by davisking** [comment]
Read the examples with "introduction" in the name.  Then read the more
advanced examples for specific application areas.  Also,
http://dlib.net/faq.html#Whereisthedocumentationforobjectfunction.  There
are thousands of paces of documentation in dlib.  If you aren't finding it
you aren't looking.

**Comment by minmummax** [comment]
oh thanks!
